Dockerfile

```diff
@@ -1,9 +1,17 @@
 FROM python:3.8-slim-buster
 
+# Install Java
+RUN apt-get update && \
+    apt-get install -y openjdk-11-jdk && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
we're working on some swtich to java17 across all repos, maybe we can upgrade this one too?

Reply:
@hweej maybe your suggestion of multistage build could be better here too